libraries/botframework-connector/src/auth/connectorFactoryImpl.ts

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { getDefaultUserAgentValue, userAgentPolicy } from '@azure/ms-rest-js';
+import { ConnectorClient } from '../connectorApi/connectorClient';
+import { ConnectorClientOptions } from '../connectorApi/models';
+import { ConnectorFactory } from '../connectorFactory';
+import { ServiceClientCredentialsFactory } from './serviceClientCredentialsFactory';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const packageInfo: Record<'name' | 'version', string> = require('../../package.json');
+export const USER_AGENT = `${packageInfo.name}/${packageInfo.version} ${getDefaultUserAgentValue()} `;
```

Comment:
This template and `require()` was ported from the `BotFrameworkAdapter` but excludes Node.js-specific values:

https://github.com/microsoft/botbuilder-js/blob/ee4705c72d49a1e421d8e3c81dddfaaf8b548401/libraries/botbuilder/src/botFrameworkAdapter.ts#L150-L157

Reply:
Note to self: need to add `Microsoft-BotFramework/3.1` to `USER_AGENT`.